src/com/goide/runconfig/testing/coverage/GoCoverageProjectData.java

```diff
@@ -160,7 +164,9 @@ public RangeData(int startLine, int startColumn, int endLine, int endColumn, int
     @Override
     public boolean equals(Object o) {
       if (this == o) return true;
-      if (!(o instanceof RangeData)) return false;
+      if (o != null && !(o instanceof RangeData)) {
```

Comment:
redundant `o != null` here. if you want to avoid possible npe, it should be separate `if`